app/controllers/users_controller.rb

```diff
@@ -14,6 +15,29 @@ def new
     @user = User.new
   end
 
+  def update
+    # @user = User.find(session[:user_id])
+    @user = User.find(params[:id])
```

Comment:
Will be changing that in the next pull request when login/logout is implemented. After discussing with Ryan, we decided this was the best choice for testing purposes until then.